tests/PopescuDan.spec.js

```diff
@@ -0,0 +1,9 @@
+// @ts-check
```

Comment:
why did you add this?

Reply:
It probably comes with the default example spec

Reply:
disregard that check.

Reply:
![image](https://github.com/adrianmaciuc/break-it-playwright-javascript-project/assets/48152758/e7d31f17-3d5d-4509-840b-49a8d50fb60d)